recipe/symfony.php

```diff
@@ -64,7 +64,7 @@
 });
 
 desc('Optimize environment variables');
-task('deploy:env', function () {
+task('deploy:env:optimize', function () {
```

Comment:
Why not using the same word?
```suggestion
task('deploy:env:dump', function () {
```